src/styles/_landing.scss

```diff
@@ -148,17 +193,30 @@
       line-height: 170%;
       margin: 8vw;
       width: 42vw;
+      @media (max-width: 700px) {
+        font-size: 3.2vw;
+      }
+      @media (max-width: 600px) {
+        font-size: 4.2vw;
+      }
+      @media (max-width: 450px) {
+        font-size: 6vw;
+        min-width: 280px;
+        width: 84vw;
+        margin-top: 25vw;
+        margin-bottom: 40vw;
+      }
       u {
         position: relative;
         text-decoration: none;
         &:after {
           content: ' ';
-          background: $peta-yellow;
+          background: #f5e401;
```

Comment:
Err yea, Han used 200 different yellows all over the chop, I tried boiling it down to a few in `variables.scss`